joss/paper.md

```diff
@@ -28,59 +28,228 @@ bibliography: paper.bib
 
 # Summary
 
-Dymos is an library for solving optimal-control type optimization problems. 
-It contains capabilities typical of optimal control software and can handle a wide range of typical optimal control problems.
-Its software design (built on top of NASA's OpenMDAO Framework [@Gray2019a] also allows users to tackle problems where the trajectory is not necessarily central to the optimization and where the system dynamics may include expensive, implicit calculations.
-Implicit calculations may be present due to some calculation in the ODE, or more generally because the ODE is posed as a set of differential algebraic equations or differential inclusions [@Seywald1994].
+Dymos is an library for optimizing control schedules for dynamic systems --- sometimes referred to as trajectory optimization. 
+There are a number of other optimal control libraries that tackle a similar kind of problem, such as OTIS4 [@Paris2006], GPOPS-II [@Patterson2014GPOPSII],and CASADI [@Andersson2018].
+Dymos can operate in a similar manner to these codes where it is focused purely on the optimal control problem, 
+but it is unique in that it can also support broader cases where trajectory optimization is just one component in a larger physical model. 
+This allows Dymos to play a critical role in solving problems where the physical design of a system is optimized simultaneously with control schedule governing its transient performance. 
+This kind of problem is often referred to as co-design, controls-co-design, or multidisciplinary design optimization.
```

Comment:
kind of feel like this sentence calls for some refs (for co-design, controls-co-design, etc), but it is up to you.